Geomar dev

seb-wahl opened the pull request.

seb-wahl committed:
added a few foci runscript examples

seb-wahl committed:
Merge branch 'release' into geomar_dev

seb-wahl committed:
added correct nest_refinement for WG10v4

mandresm committed:
remove duplicated variable

Commit:
Bump version: 6.19.4 → 6.19.5

mandresm committed:
Merge pull request #902 from esm-tools/fix/packages_permissions

fix/package_permissions

JanStreffing committed:
changed oasis branch name

I changed this branch name on the dkrz OASIS repo we use for AWI-CM3-master to destingish our main on dkrz from smhi origin main.

seb-wahl committed:
first draft of eORCA025.z75.SI3, still crashes with diamond shape patterns of NaN
during the first time step

mandresm committed:
fesom scaling tests in Albedo

mandresm committed:
Merge branch 'release' into albedo/fesom_tests

mandresm committed:
accounting true for Albedo

seb-wahl committed:
fixed two bugs in nemo postprocessing (date math)

mandresm committed:
documentation cleanup directories workaround

mandresm committed:
Merge pull request #905 from esm-tools/docs/cleanup_dir_workaround

documentation cleanup directories workaround

mandresm committed:
Merge branch 'release' into fix/oasis_branch_for_awicm3-master

Commit:
Bump version: 6.19.5 → 6.19.6

mandresm committed:
Merge pull request #904 from esm-tools/fix/oasis_branch_for_awicm3-master

changed oasis branch name

mandresm committed:
fix the default meshes conflicts between different computers for fesom-1.4 adding the mesh_base_dir variable, fix a missing block for the environment in glogin, clean useless env variables in the fesom yamls and make use of the dictionary merge for env variables in fesom.yaml

mandresm committed:
remove unused bckp namelits of fesom

seb-wahl committed:
added nemo4.2 configuration that ran 3 years using JRA-55 forcing and the v1.0.5 Levitus initial data

seb-wahl committed:
fixed date bugs in nemo_postprocessing.sh

seb-wahl committed:
added test queue on HLRN4 machines

seb-wahl committed:
Merge pull request #908 from esm-tools/feature/add-eORCA025-standalone

Feature/add eorca025 standalone

mandresm committed:
compile time fesom specifics for hlrn into general environment

seb-wahl committed:
remove set -vx

mandresm committed:
fix fesom1.4 lib problems in hlrn

mandresm committed:
fix typo

mandresm committed:
fix to mesh_base_dir in fesom1.4

JanStreffing committed:
working awicm3 on albedo

JanStreffing committed:
Merge branch 'albedo/fesom_tests' of https://github.com/esm-tools/esm_tools into albedo/fesom_tests

mandresm committed:
awicm3 running in Albedo with Miguel's libs

seb-wahl committed:
focioifs-3.0 compiles, won't run as now NEMO4.2 specific adjustments have been made
updated nemo4 example standalone runscripts.

mandresm committed:
add fesom-1.4 pools for ollie

mandresm committed:
add fesom-1.4 pools for levante. Forcing and hydrology files for fesom-1.4 are not there, but I still included the path to where they should be

Commit:
Bump version: 6.19.6 → 6.19.7

mandresm committed:
Merge pull request #906 from esm-tools/fix/fesom_hlrn

Fix/fesom hlrn

seb-wahl committed:
add _nemo4 to compile settings for glogin
uncomment compiler_mpi in blogin.yaml again

seb-wahl committed:
fix a typo in blogin.yaml

seb-wahl committed:
add TCO95_eORCA025 switch

mandresm committed:
echam works

mandresm committed:
Merge branch 'release' into feat/albedo

mandresm committed:
cleanup of albedo.yaml

pgierz committed:
awiesm 2.1 runs on albedo

Commit:
change xml files for NEMO4, and add new OpenIFS dev branch for 3.0

Commit:
Merge branch 'feature/add-focioifs-3.0' of https://github.com/esm-tools/esm_tools into feature/add-focioifs-3.0

mandresm committed:
automatic qos for albedo

mandresm committed:
Merge branch 'feat/albedo' of https://github.com/esm-tools/esm_tools into feat/albedo

mandresm committed:
fix raw restart names for the refactoring branch of FESOM

mandresm committed:
fix updating of the status in esm_tests

mandresm committed:
add testing phase warning

Commit:
Bump version: 6.19.7 → 6.19.8

mandresm committed:
Merge pull request #914 from esm-tools/fix/esm_tests_state

fix/esm_tests state update

mandresm committed:
Merge branch 'release' into feat/albedo

mandresm committed:
esm_tests switching to release branch from esm_tests_info

mandresm committed:
fix for esm_tools --test-state

pgierz committed:
fix: restore -l

Restores -l flag to shebang for now

mandresm committed:
Update esm_environment.py

fix -l in shebang

mandresm committed:
fix some changes for other computers with AWIESM

Commit:
Bump version: 6.19.8 → 6.20.0

mandresm committed:
Merge pull request #915 from esm-tools/feat/albedo

Feat/albedo

mandresm committed:
fix environment variables to use the system libs

mandresm committed:
Merge branch 'release' into fix/fesom_bin_restarts

mandresm committed:
add all the new types of parallel restarts and memory dump restart files also to fesom-2.1.yaml in case someone using that ESM-Tools version decides to switch to the refactoring branch, so that not all of them end up in the unknown and start consuming a lot of space as referred in esm-tools/esm_tools#916

Commit:
Bump version: 6.20.0 → 6.20.1

mandresm committed:
Merge pull request #916 from esm-tools/fix/fesom_bin_restarts

fix raw restart names for the refactoring branch of FESOM

mandresm committed:
add the component_template.yaml so that users that want to start adding a new component have something to start with

mandresm committed:
add a mention to the template in the documentation

JanStreffing committed:
add runscript

mandresm committed:
Merge branch 'release' into feat/awicm3-tco1279-dart

Commit:
Bump version: 6.20.1 → 6.21.0

mandresm committed:
Merge pull request #920 from esm-tools/feat/awicm3-tco1279-dart

Feat/awicm3 tco1279 dart

Commit:
add gcc 9.3.0 to main module for compiling FOCI-OpenIFS on glogin

mandresm committed:
Merge branch 'release' into fix/albedo_system_libs

Commit:
Bump version: 6.21.0 → 6.21.1

mandresm committed:
Merge pull request #917 from esm-tools/fix/albedo_system_libs

fix/albedo system libs

Commit:
Makes CLI for new templates

Commit:
docs: adds documentation for top-level commands

Commit:
new script for generating OpenIFS initial conditions from GRIB output. If all necessary fields are written in output, then this script can be used

Commit:
docs: includes type flag documentation

Commit:
bug fix for oifs processing when running annual restarts

Commit:
add LNEMOLIMCUR to run with relative winds in FOCI-OIFS

mandresm committed:
ignore connection errors from the motd

seb-wahl committed:
fixed modules for NEMO4
first modifications for NEMO4 coupled to openIFS

seb-wahl committed:
focioifs-3.0 technically up and running (piCtl)

seb-wahl committed:
Merge pull request #928 from esm-tools/feature/add-focioifs-3.0

Feature/add focioifs 3.0

seb-wahl committed:
Merge pull request #929 from esm-tools/release

merge the latest release into geomar_dev

mandresm committed:
changes the version of git required in install.sh and adds the cleanup script we used in esm_versions to easily transition to version 6 monorepo

mandresm committed:
add description to the monorepo script

Commit:
Bump version: 6.21.1 → 6.21.2

mandresm committed:
Merge pull request #931 from esm-tools/fix/monorepo_fixing_script

fix/monorepo fixing script

mandresm committed:
Merge branch 'release' into templates/component

Commit:
Bump version: 6.21.2 → 6.21.3

mandresm committed:
Merge pull request #918 from esm-tools/templates/component

templates/component

jannitzbon committed:
fixed error messages and initialization of subjob_cluster in workflow manager

mandresm committed:
Merge branch 'release' into fix/motd_no_connection

Commit:
Bump version: 6.21.3 → 6.21.4

mandresm committed:
Merge pull request #927 from esm-tools/fix/motd_no_connection

fix/motd no connection

mandresm committed:
Merge branch 'release' into fix/workflow

Commit:
Bump version: 6.21.4 → 6.21.5

mandresm committed:
Merge pull request #932 from esm-tools/fix/workflow

fixed error messages and initialization of subjob_cluster in workflow…

mandresm committed:
correct support issue template

mandresm reviewed (approved):
I'm approving this without a review since the request is not to merge to `release`

JanStreffing committed:
update io control files for IBS run

Commit:
Bump version: 6.21.5 → 6.21.6

mandresm committed:
Merge pull request #939 from esm-tools/1month_tco1279-dart

update io control files for IBS TCo1279-DART run

Commit:
add abrupt 4xCO2 scenario to oifs.yaml. Also add runscript to branch off historical and 4xCO2 experiments from FOCI-OpenIFS control

pgierz committed:
fix(database): corrects import warning for esm-database

nwieters committed:
Bugfix for wrong command example esm_tools --test-state.

nwieters committed:
Merge pull request #946 from esm-tools/bugfix_nadine_issue-945-wrong-command-for-esm_tools-test-state-in-testing-warning-message-on-albedo

Bugfix for wrong command example esm_tools --test-state.

nwieters committed:
Started to add comments to generic machine yaml file.

mandresm committed:
use admin perl modules for AWICM3

mandresm committed:
Merge branch 'release' into sqlalchemy_warning_fix

Commit:
Bump version: 6.21.6 → 6.21.7

mandresm committed:
Merge pull request #943 from esm-tools/sqlalchemy_warning_fix

fix(database): corrects import warning for esm-database

nwieters committed:
Added comments and placeholders to generic machine file.

Commit:
added eoptmized nodes for glogin

nwieters committed:
Apply suggestions from code review

Co-authored-by: Paul Gierz <pgierz@awi.de>

nwieters committed:
Apply suggestions from code review

nwieters committed:
Adapt new template to create machine file, added cookbook entry how to add a new HPC machine to esm-tools.

nwieters committed:
Implemented suggestions during review.

nwieters committed:
Added option 'machine' for argument type to create a new machine config yaml file via esm_tools create-new-config.

Commit:
fix for branch-off experiments with eternal restarts in OpenIFS. Now set ini_pseudo_initial_date in runscript for restart. Also, fix for 1pctCO2 which was not working before.

mandresm committed:
change the doc recipe for the new HPC Machine

mandresm committed:
fix typo in docs

mandresm committed:
fix more typos in docs

nwieters committed:
Apply suggestions from code review

Co-authored-by: Miguel <63242832+mandresm@users.noreply.github.com>

nwieters committed:
Merge branch 'release' into 947-comments-for-generic-maschine-file

Commit:
Bump version: 6.21.7 → 6.21.8

nwieters committed:
Merge pull request #951 from esm-tools/947-comments-for-generic-maschine-file

947 comments for generic maschine file

nwieters committed:
Update README.rst for module load ... on juwels.

nwieters committed:
Test change

mandresm committed:
Merge branch 'release' into fix/awicm3_albedo

Commit:
Bump version: 6.21.8 → 6.21.9

mandresm committed:
Merge pull request #948 from esm-tools/fix/awicm3_albedo

use admin perl modules for AWICM3

mandresm committed:
Update README.rst

mandresm committed:
Merge pull request #955 from esm-tools/954-documentation-update-in-section-before-you-continue-for-juwels

Update README.rst for module load ... on juwels.

mandresm committed:
fix problems with perl and echam in albedo

mandresm committed:
Merge branch 'fix/awicm3_albedo' of https://github.com/esm-tools/esm_tools into fix/awicm3_albedo

mandresm committed:
Merge branch 'release' into fix/echam_albedo

Commit:
Bump version: 6.21.9 → 6.21.10

mandresm committed:
Merge pull request #957 from esm-tools/fix/echam_albedo

Fix/echam albedo

JanStreffing committed:
using native grid and compressed io for tco1279-dart

JanStreffing committed:
Merge branch 'release' into 1month_tco1279-dart

Commit:
Bump version: 6.21.10 → 6.21.11

mandresm committed:
Merge pull request #958 from esm-tools/1month_tco1279-dart

1month tco1279 dart

pgierz committed:
fix: proper repr for Date objects

Commit:
Bump version: 6.21.11 → 6.21.12

nwieters committed:
Merge pull request #963 from esm-tools/calendar_repr

fix: proper repr for Date objects

nwieters committed:
Added correct oasis restart file to be copied when branching off from experiment.

mandresm committed:
trigger an error if using esm_runscripts with generic.yaml without check mode off

mandresm committed:
Update configs/machines/generic.yaml

Co-authored-by: Nadine Wieters <31928930+nwieters@users.noreply.github.com>

mandresm committed:
Update configs/machines/generic.yaml

Co-authored-by: Nadine Wieters <31928930+nwieters@users.noreply.github.com>

Commit:
Bump version: 6.21.12 → 6.21.13

mandresm committed:
Merge pull request #964 from esm-tools/fix/generic_machine_error_runtime

trigger an error if using esm_runscripts with generic.yaml without ch…

nwieters committed:
Now it should take the correct rstas.nc/rstos.nc file in case of a branchoff simulation for oasis.

nwieters committed:
Implemented review suggestions.

nwieters committed:
Implemented review suggestions.

nwieters committed:
Merge branch 'release' into 901-oasis-restarts-issue

Commit:
Bump version: 6.21.13 → 6.21.14

nwieters committed:
Merge pull request #967 from esm-tools/901-oasis-restarts-issue

901 oasis restarts issue

pgierz committed:
fix(echam.yaml): updates online handbook with correct licence guidelines

nwieters committed:
Added selection of oasis restart file if more than one is found.

nwieters committed:
Merge pull request #970 from esm-tools/docs/echam_docs

fix(echam.yaml): updates online handbook with correct licence guidelines

nwieters committed:
Added review suggestions.

nwieters committed:
Changed the questinary part for oasis restart files so it is be a bit shorter.

pgierz committed:
fix(rtd): fails for any warning while building documentation

Commit:
openifs runs on NESH, but post fails with git exec error

mandresm committed:
add minor changes to the comments and fix an if statement for setting the ini_restart_dir variable

nwieters committed:
Canged 'if lresume is False' to 'if lresume == False'.

nwieters committed:
Canged 'if lresume is False' to 'if lresume == False'; broke down long text strings into multiple lines

nwieters committed:
Bugfix in oasis.py if lresume is ...

pgierz committed:
Update src/esm_runscripts/oasis.py

pgierz committed:
Update src/esm_runscripts/oasis.py

pgierz committed:
Merge pull request #975 from esm-tools/fix/lresume_oasis

Bugfix in oasis.py if lresume is ...

Commit:
oifs-43r3-v2 is now based on branch foci211. Also fixed postprocessing for AMIP runs

mandresm committed:
Bump version: 6.21.14 → 6.21.15

mandresm committed:
Merge branch 'fix/lresume_oasis' into release

mandresm committed:
Merge branch 'release' into fix/969-choose-oasis-restart-file-if-not-unambiguously-clear

mandresm committed:
cleanup oasis.py

Commit:
new xml files

pgierz committed:
fix(scope): correct the url

pgierz committed:
fix(scope): fixes url of git repo with git ending

pgierz committed:
fix(esm_master/task.py): allow pip-based installed 'extra' packages to install in venvs

Commit:
Bump version: 6.21.15 → 6.21.16

pgierz committed:
Merge pull request #980 from esm-tools/fix-esm-master-venv-install

fix(esm_master/task.py): allow pip-based installed 'extra' packages t…

pgierz committed:
Merge branch 'fix-esm-master-venv-install' into scope-fix-remote

mandresm committed:
fix a bug that was killing esm_master

mandresm committed:
fix small bug in oasis.py

mandresm committed:
add motd for the oasis branchoff restart fix

mandresm committed:
Merge branch 'release' into fix/969-choose-oasis-restart-file-if-not-unambiguously-clear

Commit:
Bump version: 6.21.16 → 6.21.17

mandresm committed:
Merge pull request #971 from esm-tools/fix/969-choose-oasis-restart-file-if-not-unambiguously-clear

Added selection of oasis restart file if more than one is found.

pgierz committed:
Merge branch 'release' into scope-fix-remote

Commit:
Bump version: 6.21.17 → 6.21.18

pgierz committed:
Merge pull request #979 from esm-tools/scope-fix-remote

fix(scope): correct the url

mandresm committed:
update motd

Commit:
add orbital parameters to climMEMODEL runscript

nwieters committed:
Bugfix for selction of oasis restart file.

nwieters committed:
Bugfix for oasis restarts in branchoff exp.

Commit:
Bump version: 6.21.18 → 6.21.19

mandresm committed:
Merge pull request #982 from esm-tools/fix/oasis_select_restart

Bugfix for selction of oasis restart file.

seb-wahl committed:
Merge pull request #983 from esm-tools/release

update geomar_dev with latest release

Commit:
module load git git-lfs on nesh

Commit:
Merge branch 'geomar_dev' of https://github.com/esm-tools/esm_tools into geomar_dev

Commit:
set ini_pseudo_initial_date in case not already set

Commit:
add L75 option. Also add Calving mask for AGRIF, e.g. ORION10

Commit:
add Intel 2022 / IMPI 2021 as compile option on glogin. The older, Intel 2019, stopped working

Commit:
add settings for calving on AGRIF, and also Tco319-ORCA12 config. Also set default compiler on glogin to Intel 2022/IMPI 2021

Commit:
runscript to run Tco319-ORCA12 on glogin. Works, but slower than Tco95 version. Likely due to high I/O from XIOS

Commit:
Merge branch 'geomar_dev' of https://github.com/esm-tools/esm_tools into geomar_dev

Commit:
update IO_LIB_ROOT for Intel 2022 compilers on glogin

Commit:
New Intel compilers (2022 and beyond) do not accept -mkl=sequential anymore. Replace by -qmkl=sequential, which should work on any new-ish versions.

Commit:
fix so that clean command also removed BLD and arch_nemo.fcm for NEMO.

Commit:
fixed so that month 08 is no longer interpreted as an octal number. Should only impact names of postprocessed OpenIFS output.

Commit:
add OpenIFS output on theta levels to oifs_postprocessing.sh

joakimkjellsson committed:
switch domain_cfg file for NEMO4. The old one was bad

joakimkjellsson committed:
add gcc so that XIOS compiles properly

joakimkjellsson committed:
bugfix so that nemo post script works for monthly restarts properly. 08 was being interpreted as octal numbers...

joakimkjellsson committed:
better default nproc settings for FOCI-OpenIFS 4.0

joakimkjellsson committed:
make new version: FOCI-OpenIFS 2.2 with new switches to use 1st order conserv remapping with oasis

seb-wahl committed:
first draft of fs_oasismct4
cleanup still required

joakimkjellsson committed:
introduce FOCI-OpenIFS v2.2 using OASIS3-MCT master branch directly from CERFACS

joakimkjellsson committed:
add clean command for oasis mct5

joakimkjellsson committed:
add file_def_agrif for agrif-3.0

joakimkjellsson committed:
we can now choose conserv (1st order) or gauswgt for atm-agrif remapping. conserv does not work though

joakimkjellsson committed:
new way to activate load balancing tool (LUCIA) in OASIS MCT5

joakimkjellsson committed:
added FOCI-OpenIFS 3.0_busan. OpenIFS 43r3 + NEMO 4.2 (ORCA05 L46)

joakimkjellsson committed:
added FOCI-OpenIFS 2.2 with OASIS-MCT5. Works on glogin

joakimkjellsson committed:
add correct xml files for nemo4 (focioifs 3.0_busan)

joakimkjellsson committed:
Merge pull request #1101 from esm-tools/feature/oasis_from_cerfacs

Feature/oasis from cerfacs

joakimkjellsson committed:
add variable oasis_p to control if we set P 2 (for NEMO 3.6) or P 0 (for NEMO 4) in namcouple file. This controls if periodic points are included in the coupling fields or not

joakimkjellsson committed:
switch to OASIS MCT5 for Busan version of FOCI-OpenIFS

joakimkjellsson committed:
change MCT version also here

joakimkjellsson committed:
add bilinear and bicubic remapping methods for FOCI-OpenIFS. This means we handle everything as bilinear but use bicubic for wind stress

joakimkjellsson committed:
output contains z on surface, not hybrid level

joakimkjellsson committed:
updated runscript for focioifs3_busan

joakimkjellsson committed:
change leapyear to true for ORCA05 coupled config with NEMO4. Warning: If leapyear is false in NEMO coupled to OpenIFS, ESM-Tools and OASIS wont complain, but XIOS will throw strange errors after 3 years

seb-wahl committed:
Merge pull request #1110 from esm-tools/focioifs3_busan

Focioifs3 busan

joakimkjellsson committed:
switch domain_cfg for NEMO4-ORCA05 to exclude Caspian Sea

joakimkjellsson committed:
Switch NEMO4 version to NEMO 4.2.2 (ice drag bugfix)

joakimkjellsson committed:
switch to intel 2022 on glogin. seems fine and its good to be up to date

joakimkjellsson committed:
add NEMO4 output to postprocessing. diaptr2D etc

joakimkjellsson committed:
add ICMSH and ICMGG files to output files for OpenIFS

joakimkjellsson committed:
update runscripts for FOCI-OpenIFS

joakimkjellsson committed:
update runscripts for FOCI-OpenIFS

joakimkjellsson committed:
Merge pull request #1131 from esm-tools/focioifs3_busan

Focioifs3 busan

joakimkjellsson committed:
added regular_pl_zoom output for OpenIFS, which is pressure-level output but only for 850, 500 and 300 hPa. Useful to output high-frequency fields while keeping data size down

joakimkjellsson committed:
added runscripts for HighResMIP runs with FOCI-OpenIFS-VIKING10

seb-wahl committed:
added (I know it's the wrong branch)
- gauswght direct coupling for FOCI
- a few missing runscripts
- bugfix for JSBACH for hist runs

seb-wahl committed:
removed -l from default shell interpreter, caused problems on HLRN
fix search pattern for restart file naming convention used with FOCI

seb-wahl committed:
Merge pull request #1132 from esm-tools/feat/foci_mct4_smag

Feat/foci mct4 smag
